src/main/resources/reference.conf

```diff
@@ -1,41 +1,127 @@
 # ===================
 # Redis cache configuration
 # ===================
-play.cache.redis {
-  # redis server: host
-  host:       localhost
-  # redis server: port
-  port:       6379
-  # redis server: database number
-  database:   1
-  # list of nodes in a cluster
-  cluster:    [
-    # nodes are defined as a sequence of objects:
-    #  required string, defining a host the node is running on
-    #  host:        localhost
-    #  required integer, defining a port the node is running on
-    #  port:        6379
-    #  optional string, defines a password to use
-    #  password:    null
-  ]
-  # connection timeout in milliseconds
-  timeout:    1s
+play.cache.redis: {
+
+  # list of named caches available in the application
+  #
+  # for more details see https://www.playframework.com/documentation/2.6.x/ScalaCache#Accessing-different-caches
+  #
+  instance {
+
+    # definition of the default cache,
+    # its name is defined in 'play.cache.defaultCache'
```

Comment:
`play.cache.redis.default-cache` is defined below, but `play.cache.defaultCache` is mentioned here